tests/dummy/app/pods/view/renderers/models/checkbox-array.js

```diff
@@ -6,6 +6,19 @@ export default {
         type: 'string'
       },
       type: 'array'
+    },
+    serviceAffecting: {
+      items: {
+        type: 'string'
```

Comment:
I'm pretty sure this also needs an `enum: ['SERVICE_AFFECTING', 'NOT_SERVICE_AFFECTING']` to be accurate.